Countries/ListViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ListViewController.swift
+//  Countries
+//
+//  Created by Khrystyna Matasova on 23.11.2021.
+//
+
+import UIKit
+
+class ListViewController: UIViewController {
```

Comment:
Why do you ignore UITableViewController?

Reply:
It looks easier for me, but if I need to do with TableViewController - ok